cranelift/codegen/src/isa/x64/inst.isle

```diff
@@ -316,6 +316,11 @@
        ;; A pseudoinstruction that just keeps a value alive.
        (DummyUse (reg Reg))
 
+       ;; A pseudoinstruction that loads the address of an exception
+       ;; handler target.
+       (ExceptionHandlerAddress (dst WritableGpr)
+                                (label MachLabel))
+
```

Comment:
As a mild bikeshed these can probably be named something like "load the label address in this register" since it's not really relate to exceptions. I'm a bit sad a new MInst was needed everywhere but on-the-whole this is a great change!

Reply:
Done!